doc/developer/mzbuild.md

```diff
@@ -462,6 +462,14 @@ COPY --from=0 ...
   command][dockerfile-from], except that the image named must be a valid mzbuild
   image in the repository, not a vanilla Docker image.
 
+### Environment Variables
+
+The following environment variables are supported by all commands:
+
+* `BUILD_MODE` can be set to `release` (the default) or `debug`. If Cargo
```

Comment:
minor nit: the rust name for the faster, development build profile is `dev`, and not `debug` -- see https://doc.rust-lang.org/cargo/reference/profiles.html#dev

The main reason I think this is important is because we use `dev` in other parts of the code, so having both `debug` and `dev` options for various things is confusing

Reply:
TIL. I used `debug` because I think that's the directory name where cargo dumps the output binary, and part of the code uses that directory name to copy the binary from. Options seem to be:

1) Document that `debug` is the `dev` profile, but don't change the current code.
2) Support `dev` and `release`, but have some code that maps `dev` to `debug` when looking through paths.

I'm lightly in favor of 1?

Reply:
I don't have a strong preference here -- I think its fine to pick whatever you like more for now and maybe check with Nikhil when he's back since he's had opinions on this sort of thing in the past?